skate-plugin/src/main/resources/messages/skateBundle.properties

```diff
@@ -8,6 +8,7 @@ skate.configuration.enableProjectGenMenuAction.title=Enable Project Gen Menu Act
 skate.configuration.enableProjectGenMenuAction.description=If enabled, shows Slack Project Generator option under Tools menu
 skate.configuration.enableFeatureFlagLinking.title=Enable Feature Flag Linking
 skate.configuration.enableFeatureFlagLinking.description=If enabled, show inspections for opening feature flag links from corresponding enum entries.
+skate.configuration.featureFlagFilePatterns.title=Provide a list of comma-separated file name patterns (e.g. Feature.kt) for feature flag checking
```

Comment:
Specify if it's a regex, glob, suffix, etc. judging from the code it's a comma-separated list of suffixes, which isn't obvious from this labeling. Let also doc the settings property in code. Usually we use regexes or globs for flexibility, both of which are supported by the JDK.

Reply:
[f1a16cb](https://github.com/slackhq/slack-gradle-plugin/pull/601/commits/f1a16cb08b3d32c07a33dcd3f126483c6e9b0a99)